src/main/java/com/minecraftdimensions/bungeechatfilter/Rule.java

```diff
@@ -43,81 +41,108 @@ public String getStringRegex() {
         return regex.pattern();
     }
 
-    public Matcher getMatcher( String message ) {
-        return regex.matcher( message );
+    public Matcher getMatcher(String message) {
+        return regex.matcher(message);
     }
 
-    public boolean doesMessageContainRegex( String message ) {
-        return getMatcher( message ).find();
+    public boolean doesMessageContainRegex(String message) {
+        return getMatcher(message).find();
     }
 
-    public void performActions( ChatEvent event, ProxiedPlayer player ) {
+    public void performActions(ChatEvent event, ProxiedPlayer player) {
         String message = event.getMessage();
-        if(ignore!=null){
-            Matcher ig =Pattern.compile(ignore.pattern()).matcher(message);
-                while(ig.find()){
+        if (ignore != null) {
+            Matcher ig = Pattern.compile(ignore.pattern()).matcher(message);
+            while (ig.find()) {
                 return;
             }
         }
-        for ( String action : actions.keySet() ) {
-            if ( action.equals( "deny" ) ) {
-                event.setCancelled( true );
-            } else if ( action.equals( "message" ) ) {
-                player.sendMessage( TextComponent.fromLegacyText( Main.color( actions.get( action )[0] ) ) );
-            } else if ( action.equals( "kick" ) ) {
-                player.disconnect( new TextComponent( TextComponent.fromLegacyText( Main.color( actions.get( action )[0] ) ) ) );
-            } else if ( action.equals( "alert" ) ) {
-                String alert =   actions.get( action )[0].replace( "{player}", player.getDisplayName() );
-                if(message.split( " ", 2 ).length>1){
-                       alert =alert.replace("{arguments}", message.split( " ", 2 )[1] )    ;
+        for (String action : actions.keySet()) {
+            if (action.equals("deny")) {
```

Comment:
I can see that these lines at the top have just been reformatted again, which makes it difficult to easily read the actual changes made in this block of changed lines (and we don't want you reformatting the code in this PR)